README.md

```diff
@@ -1,126 +1,174 @@
+This function is available on the `NonFungibleToken.CollectionPublic` interface,
+which accounts publish as public capability. 
+This capability allows anybody to deposit an NFT into a collection 
+without accessing the entire collection.
 
-#### 8 - NFTCollection Resource Destructor
+```swift
+let nft: ExampleNFT.NFT
 
-- no event
+// ...
 
-## Comparison to other Standards on Ethereum
+let collection = account.getCapability(/public/ExampleNFTCollection)
+    .borrow<&{NonFungibleToken.CollectionPublic}>()
+    ?? panic("Could not borrow a reference to the receiver's collection")
+            
+collection.deposit(token: <-nft)
+```
 
-This covers much of the same ground that a spec like ERC-721 or ERC-1155 covers, but without most of the downsides.  
+#### ⚠️ Important
 
-- Tokens cannot be sent to contracts that don't understand how to use them, because an account has to have a `Receiver` or `Collection` in its storage to receive tokens.
-- If the recipient is a contract that has a stored `Collection`, the tokens can just be deposited to that Collection without having to do a clunky `approve`, `transferFrom`
-- Events are defined in the contract for withdrawing and depositing, so a recipient will always be notified that someone has sent them tokens with their own deposit event.
-- This version can support batch transfers of NFTs. Even though it isn't explicitly defined in the contract, a batch transfer can be done within a transaction by just withdrawing all the tokens to transfer, then depositing them wherever they need to be, all atomically.
-- Transfers can trigger actions because users can define custom `Receivers` to execute certain code when a token is sent.
-- Easy ownership indexing:  Instead of having to iterate through all tokens to find which ones you own, you have them all stored in your account's collection and can get the list of the ones you own instantly
+In order to comply with the deposit function in the interface,
+an implementation MUST take a `@NonFungibleToken.NFT` resource as an argument. 
+This means that anyone can send a resource object that conforms to `@NonFungibleToken.NFT` to a deposit function. 
+In an implementation, you MUST cast the `token` as your specific token type before depositing it or you will 
+deposit another token type into your collection. For example:
+
+```swift
+let token <- token as! @ExampleNFT.NFT
+```
+
+### List NFTs in an account
+
+Return a list of NFTs in a `Collection` using the [`getIDs`](contracts/ExampleNFT.cdc#L59-L62) function.
+
+This function is available on the `NonFungibleToken.CollectionPublic` interface,
+which accounts publish as public capability. 
+
+```swift
+let collection = account.getCapability(/public/ExampleNFTCollection)
+    .borrow<&{NonFungibleToken.CollectionPublic}>()
+    ?? panic("Could not borrow a reference to the receiver's collection")
+    
+let ids = collection.getIDs()
+```
+
+### Read an NFT from an account
```

Comment:
might be useful to just remove this section completely since the borrowNFT function is useless. It might give people the wrong idea